src/servases/XfilesApi.jsx

```diff
@@ -0,0 +1,14 @@
+const getXFilesMain = () => {
+  return fetch('https://xfiles-api.herokuapp.com/api/v1/characters?category=Main_characters', {
+  })
```

Comment:
You don't need to create this object at the end of your fetch call if you're not adding options. Could just be:
const getXFilesMain = () => {
  return fetch('https://xfiles-api.herokuapp.com/api/v1/characters?category=Main_characters', {
  })
 

 
You don't need to create this object at the end of your fetch call if you're not adding options. Could just be:
const getXFilesMain = () => {
  return fetch('https://xfiles-api.herokuapp.com/api/v1/characters?category=Main_characters')
    .then(res => res.json());
};